### DISCUSSION TITLE
Please Give Better Feed Configurations

### DISCUSSION THREAD

**Discussion by atom0s** [community/community#37476] [Other Feature Feedback, Questions, & Ideas]
The new feed is honestly useless. It is completely undoing any kind of productive aspect the homepage used to have. For me, I personally used the old page set up to follow my own project activity. (Stars, forks, new issues, etc.) I don't care for the notification page so I used the feed page instead to keep up to date.

I follow 1 person who does not post almost anything on this site anymore. Yet, the new feed for me is full of all kinds of people I don't know and don't follow. I only have 2 repositories starred that aren't mine, yet I get force-fed information about a ton of repos I have never seen, heard of, used, starred/forked, followed/watched, etc. 

Please give us the option to only see things that actually matter to us. I don't use GitHub to 'connect' with others. This is not a social media platform. I understand you want more collaboration and some users may find these features useful as they have been [rarely] requested, but myself and the majority don't want it to be the main/forced normal.

I suggest that the following changes be added or available to be selected:

  - Hide activity of other users that I am not following.
  - Hide activity of other repositories that I am not following/watching.
  - Hide activity for collaborated repos if the activity is not related to my collaboration. (ie. Only show me updates to issues/pulls I am watching, not the entire repos activity.)

I would prefer my homepage feed to only show me the following:

  - New issues / pull requests on any of my personal repos.
  - New sponsors to me or my projects.
  - New follows to me.

I do not want to see anything else personally. Again, I don't use GitHub as a social media platform. I am not here to make friends or 'chat' with others.

**Comment by ossanna16** [comment]
Hi there @atom0s and welcome to our community! Thank you for the feedback 🙂
